Homework13/PageObjects/WebTablePage.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Text;
+using Homework13.Common.WebElements;
+using OpenQA.Selenium;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Homework13.PageObjects
+{
+    public class WebTablePage
+    {
+        private DemoQaWebElement _addNewEntryButton = new(By.XPath("//button[@id='addNewRecordButton']"));
+        private DemoQaWebElement _submitButton = new(By.XPath("//button[@id='submit']"));
+        private DemoQaWebElement _form = new(By.XPath("//form"));
+        private DemoQaWebElement _firstName1 = new(By.XPath("((//div[@role='rowgroup'])[1]//descendant::div[@role='gridcell'])[1]"));
+        private DemoQaWebElement _tableRows = new(By.XPath("//div[@class='rt-tbody']"));
+        private DemoQaWebElement _searchTextBox = new(By.XPath("//input[@id='searchBox']"));
+        private DemoQaWebElement _firstNameTextBox = new(By.XPath("//input[@id='firstName']"));
+        private DemoQaWebElement _lastNameTextBox = new(By.XPath("//input[@id='lastName']"));
+        private DemoQaWebElement _emailTextBox = new(By.XPath("//input[@id='userEmail']"));
+        private DemoQaWebElement _ageTextBox = new(By.XPath("//input[@id='age']"));
+        private DemoQaWebElement _salaryTextBox = new(By.XPath("//input[@id='salary']"));
+        private DemoQaWebElement _departmentTextBox = new(By.XPath("//input[@id='department']"));
+        private DemoQaWebElement _deleteButton = new(By.XPath("//span[@id='delete-record-1']"));
+        private DemoQaWebElement _closeButton = new(By.XPath("//span[contains(text(),'×')]"));
+
+        public bool IsFormValidated() => _form.GetClass().Contains("was-validated");
+
+        public void ClickAddNewEntryButton() => _addNewEntryButton.Click();
+
+        public void ClickCloseButton() => _closeButton.Click();
+
+        public void ClickSubmitButton() => _submitButton.Click();
+
+        public void ClickEditEntryButton(string firstName) => _editButton(firstName).Click();
+
+        public void ClickDeleteEntryButton() => _deleteButton.Click();
```

Comment:
Not clear which entry will be deleted. Improve method to take index of row that should be deleted